### PR TITLE
MYST-477 connection cancelling

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

File: src/libraries/api/errors.js [hunk 1]
```diff
@@ -15,5 +17,6 @@ function hasHttpStatus (err, expectedStatus) {
 export {
   isTimeoutError,
   hasHttpStatus,
-  CONNECTION_ABORTED_ERROR_CODE
+  CONNECTION_ABORTED_ERROR_CODE,
+  CLOSED_REQUEST_CODE
```
[the remark this answers is not in the document]
Comment (Waldz) [reply src/libraries/api/errors.js]: Lets agree on same naming, lets call this feature in all repos the same.
In Node it's "cancelConnection" not *closeRequest*, @tadovas what do You think?

[the remark this answers is not in the document]
Comment (donce) [reply src/libraries/api/errors.js]: @Waldz this code does not represent any feature - it represents HTTP status code 499, which stands for `CLIENT CLOSED REQUEST`: https://httpstatuses.com/499

**Comment by donce** [comment]
@tadovas @Waldz pushed refactoring to clarify 499 status code - it's HTTP meaning and our meaning are preserved.

File: src/libraries/api/errors.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Waldz) [reply src/libraries/api/errors.js]: Lets see if it will be clear in real life

**Pull request merged**